### ISSUE TITLE
Dismiss on Action

### ISSUE THREAD

**Issue by BalazsSzamody** [AndreaMiotto/PartialSheet#9]
Hi,

Nice work.

Just a suggestion:
It would be nice to be able to dismiss the modal from code the same way as a stock modal view. With:

```Swift
@Environment (\.presentationMode) var presentationMode
```

and then:

```Swift
{
  self.presentationMode.wrappedValue.dismiss()
}
```

Thanks in advance if you consider it.

**Comment by AndreaMiotto** [comment]
Great Idea, I will add it in the next release! Stay tuned

**Comment by AndreaMiotto** [comment]
released version 2.0 with Environment object included. Give it a try.
